web-client/public/gene/api.js

```diff
@@ -358,26 +361,31 @@ let parseJaspar = function (data) {
                defaultValues.ncbi = parseNCBI(ncbiInfo);
                return window.api.getUniProtInfo(symbol);
            }).then(function (uniProtInfo) {
+            //    console.log("in uniprot")
                defaultValues.uniprot = parseUniprot(uniProtInfo);
+               return window.api.getJasparInfo(symbol);
+           }).then(function (jasparInfo) {
+            //    console.log("this is jasparInfo: ")
+            //    console.log(jasparInfo)
+               defaultValues.jaspar = parseJaspar(jasparInfo);
                return window.api.getYeastMineInfo(symbol);
            }).then(function (yeastMineInfo) {
+            //    console.log("in yeastmine")
                defaultValues.sgd = parseYeastmine(yeastMineInfo);
             //    return window.api.getFlyMineInfo(symbol);
         //    }).then(function (flyMineInfo) {
             //    defaultValues.sgd = parseFlymine(flyMineInfo);
                return window.api.getGeneOntologyInfo(symbol);
            }).then(function (goInfo) {
+            //    console.log("inside GO call")
                defaultValues.geneOntology = parseGeneOntology(goInfo);
                return window.api.getRegulationInfo(symbol);
            }).then(function (regulationInfo) {
                // parseRegulators needs both info and symbol
                defaultValues.regulators = parseRegulators(regulationInfo, symbol);
-               return window.api.getJasparInfo(symbol);
-           }).then(function (jasparInfo) {
-               defaultValues.jaspar = parseJaspar(jasparInfo);
                return defaultValues;
            }).catch(function () {
-               window.api.getNCBIInfo(symbol);
+            //    window.api.getNCBIInfo(symbol);
```

Comment:
I realize you’ve been adding these to help debug, so just remember to delete them fully once you’re all done.